addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -270,7 +270,7 @@ export class MassMailingHtmlField extends HtmlField {
             });
             this.mobilePreview = this.dialog.add(MassMailingMobilePreviewDialog, {
                 title: _t("Mobile Preview"),
-                preview: mailingHtml.body.innerHTML,
+                preview: markup(mailingHtml.body.innerHTML),
```

Comment:
@odoo/rd-security : Add markup here to make the preview safe.

Reply:
In this case mailingHtml.body.innerHTML is safe

Reply:
so we can markup it